README.md

```diff
@@ -13,134 +13,234 @@
+[Data Science: Wrangling](https://www.edx.org/course/data-science-wrangling) | 1 week | 15 hours/week | - | 
 
-### Data Science Tools & Methods
-[Tools for Data Science](https://www.coursera.org/learn/open-source-tools-for-data-science)
 
-[Data Science Methodology](https://www.coursera.org/learn/data-science-methodology)
+## Machine Learning/Data Mining
+Machine learning is the science of getting computers to act without being explicitly programmed. In the past decade, machine learning has given us self-driving cars, practical speech recognition, effective web search, and a vastly improved understanding of the human genome. this courses will cover introduction to machine learning, datamining, statistical pattern recognition and text Mining of Massive Datasets.
 
-[Data Science: Wrangling](https://www.edx.org/course/data-science-wrangling)
+**Topics covered**:
+`Logistic Regression`
+`Artificial Neural Network`
+`Decision Trees`
+`Feature Scaling`
+`Clustering`
+`SQL`
+`SQL`
+`and more`
 
-### Machine Learning/Data Mining
-[Machine Learning](https://www.coursera.org/learn/machine-learning)
+Courses | Duration | Effort | Prerequisites | Discussion
+:-- | :--: | :--: | :--: | :--:
+[Machine Learning](https://www.coursera.org/learn/machine-learning) | 4 weeks | 15 hours/week | - | 
+[Intro to Machine Learning](https://www.udacity.com/course/intro-to-machine-learning--ud120) | 10 weeks | 10 hours/week | - | 
+[Mining Massive Datasets](https://www.edx.org/course/mining-massive-datasets) | 3 weeks | 15 hours/week | - | 
+[Process Mining](link) | 4 weeks | 7-9 hours/week | - | 
 
-[Intro to Machine Learning](https://www.udacity.com/course/intro-to-machine-learning--ud120)
 
-[Mining Massive Datasets](https://www.edx.org/course/mining-massive-datasets)
 
-[Process Mining](https://www.coursera.org/learn/process-mining)
+# How to use this guide
 
-## How to use this guide
 
 ### Duration
-It is possible to finish within about 2 years if you plan carefully and devote roughly 20 hours/week to your studies. Learners can use [this spreadsheet](https://docs.google.com/spreadsheets/d/1TEGSUQDFuWL3TYNjiM8G3esly-tKOcgHSDABt92mzdA/copy) to estimate their end date. Make a copy and input your start date and expected hours per week in the `Timeline` sheet. As you work through courses you can enter your actual course completion dates in the Curriculum Data sheet and get updated completion estimates.
+It is possible to finish within about 2 years if you plan carefully and devote roughly 20 hours/week to your studies. 
+
 
 ### Order of the classes
 
 Some courses can be taken in parallel, while others must be taken sequentially. All of the courses within a topic should be taken in the order listed in the curriculum. The graph below demonstrates how topics should be ordered.
 
 ![Topic Progression Graph](topic_progression_graph.jpg)
 
-### Track your progress
```

Comment:
It's unclear why you've removed links to resources that have been created by other OSSUnians.

Reply:
because i edited the Algorithms section to make it like the CS curriculum, so they are slightly different.